HelloWorld.cs

```diff
@@ -0,0 +1,345 @@
+                attr == 'C' ? ConsoleColor.Cyan :
+                ConsoleColor.White;
+            
+            Console.ForegroundColor = color;
+            Console.Write(symbol);
+        }
+
+        Console.ResetColor();
+        Console.WriteLine();
+    }
+
+    private static void DrawMac()
+    {
+        Console.WriteLine("Hello, Mac...");
+
+        const string Apple = @"
+                                  @@@@ 
+                               @@@@@@
+                             @@@@@@@@
+                           @@@@@@@@@
+                          @@@@@@@@@
+                         @@@@@@@@
+                          @@@@
+                          
+          @@@@@@@@@@@         @@@@@@@@@@@@@
+       @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+     @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+   @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+ @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+   @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+    @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+     @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+      @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+       @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+         @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+           @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
+             @@@@@@@             @@@@@@@";
+
+
+        foreach (char c in Apple)
+        {
+            if (c == '@')
+            {
+                Console.BackgroundColor = ConsoleColor.White;
+                Console.Write(" ");
+            }
+            else
+            {
+                Console.ResetColor();
+                Console.Write(c);
+            }
+
+        }
+
+        Console.ResetColor();
+        Console.WriteLine();
+    }
+
+    //HACK: Needed until CoreFX implements Environment.OSVersion
+    private static string DetectOS()
+    {
+        Process proc = new Process();
+
+        string detectionResult = String.Empty;
+
+        if (proc.StartInfo.Environment.ContainsKey("TERM_PROGRAM"))
+        {
+            detectionResult = "mac";
+        } else if (proc.StartInfo.Environment.ContainsKey("SHELL"))
+        {
+            string shellValue = proc.StartInfo.Environment["SHELL"].ToLowerInvariant();
+
+            if (shellValue.EndsWith("/tcsh") || shellValue.EndsWith("/sh"))
+            {
+                detectionResult = "freebsd";
+            } else {
+                detectionResult = "linux";
+            }
+        } else if (proc.StartInfo.Environment.ContainsKey("COMSPEC"))
```

Comment:
The bracing on the else/else-ifs should be
```
if (condition)
{
    //body;
}
else if (condition2)
{
    //body2;
}
else
{
    //body3;
}
```

https://github.com/dotnet/corefx/blob/master/Documentation/coding-style.md (rule 1)